exporters/user_events/src/utils.cc

```diff
@@ -0,0 +1,87 @@
+
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/exporters/user_events/logs/utils.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+
+namespace exporter
+{
+namespace user_events
+{
+namespace utils
+{
+
+namespace api_common = opentelemetry::common;
+
+const int kAttributeValueSize = 16;
+
+void PopulateAttribute(nostd::string_view key,
+                       const api_common::AttributeValue &value,
+                       ehd::EventBuilder &event_builder) noexcept
+{
+  static_assert(nostd::variant_size<api_common::AttributeValue>::value == kAttributeValueSize,
+                "AttributeValue has changed, update PopulateAttributeValue");
+
+  const char *key_name = key.data();
+
+  if (nostd::holds_alternative<bool>(value))
+  {
+    event_builder.AddValue(key_name, nostd::get<bool>(value), event_field_format_default);
```

Comment:
event_field_format_boolean

Reply:
Thanks for the catch. Just curious, will it make difference to the serialization process with `event_field_format_default`?

Reply:
Yes. For an 8-bit value, default means "Treat as unsigned integer" (1/0) and bool means "treat as Boolean" (true/false).